deploy/notary.go

```diff
@@ -84,31 +84,19 @@ func enableNotary(ctx context.Context, prm enableNotaryPrm) error {
 		}
 	}
 
-	roleContract := rolemgmt.NewReader(invoker.New(prm.blockchain, nil))
```

Comment:
decided to init it again and again?

Reply:
it only does `return &ContractReader{invoker}` inside but ok